et/docker.ts

```diff
@@ -30,7 +30,7 @@ export async function destroyEverything() {
  */
 async function ccdLogin() {
   temporaryLog('Running ./ccd login')
-  return execCommand('./ccd login', process.env.ECM_DOCKER_DIR)
+  return await execCommand('./ccd login', process.env.ECM_DOCKER_DIR)
```

Comment:
Why does await need to be there? It returns a promise either way.

Reply:
I think it only matters if you want to know the result during the execution of the async function. That is, if for example there's a try catch block or if the value returned (as a new promise) depends on the async call's resolved value.

Reply:
Its an ESLint rule, makes stack traces nicer to read if theres an error (https://typescript-eslint.io/rules/return-await/)